prow/getting_started_deploy.md

```diff
@@ -16,9 +16,14 @@ You need a few things:
 1. The prow `tackle` utility. It is recommended to use it by running `bazel run //prow/cmd/tackle` from `test-infra` directory, alternatively you can install it by running `go get -u k8s.io/test-infra/prow/cmd/tackle` (in that case you would also need go installed and working).
 1. A [personal access token][1] for the GitHub bot user you want to act as prow. If you're setting up prow for personal repo (not organization) then bot user == you (your GitHub account).
```

Comment:
Ah, how did this sneak in? This is directly contradictory to what we recommend in the `Bot account` section below: 
> Note that events triggered by this account are
ignored by some prow plugins. It is prudent to use a different bot account for
other Github automation that prow should interact with to prevent events from
being ignored unjustly.

This is a very common problem for first time Prow users so we should be sure to recommend that they create a separate bot account for Prow.

Reply:
I'm going to pull the "bot account" stuff out of the two different paths and put it up top.